packages/nx-plugin/src/utils/testing-utils/async-commands.ts

```diff
@@ -10,15 +10,17 @@ import { getPackageManagerCommand } from '@nrwl/devkit';
  */
 export function runCommandAsync(
   command: string,
-  opts = {
+  opts: { silenceError?: boolean; env?: NodeJS.ProcessEnv } = {
     silenceError: false,
+    env: {},
   }
 ): Promise<{ stdout: string; stderr: string }> {
   return new Promise((resolve, reject) => {
     exec(
       command,
       {
         cwd: tmpProjPath(),
+        env: { ...process.env, ...(opts.env ? opts.env : {}) },
```

Comment:
You can spread null, so this should just be `...opts.env`. 
```suggestion
        env: { ...process.env, ...opts.env},
```